setup.py

```diff
@@ -38,8 +38,10 @@ def get_extras_require():
             results[key] = parsed
 
     lite = results.pop("lite")
+    contrib = results.pop("contrib")
```

Comment:
This will throw if `contrib` is not present but we probably do want to die in that case.

Reply:
The requirements-dev-contrib.txt file exists, so that will never raise an exception.